internal/test/wmcb/wmcb_test.go

```diff
@@ -54,6 +55,16 @@ const (
 	hybridOverlayConfigurationTime = 2 * time.Minute
 	// hybridOverlayServiceName is the name of hybrid-overlay Windows service
 	hybridOverlayServiceName = "hybrid-overlay-node"
+	// containerdDir is the directory for storing the containerd files in the payload directory
+	containerdDir = payloadDirectory + containerdServiceName
+	// winContainerdDir is the directory where the containerd files are placed in remote directory
+	winContainerdDir = remoteDir + containerdServiceName
+	//containerdServiceName is containerd Windows service name
+	containerdServiceName = "containerd"
```

Comment:
nit, move `containerdServiceName` definition above `containerdDir` and `winContainerdDir` which use it